src/release_cli.rs

```diff
@@ -1,14 +1,24 @@
-use crate::{changelog, config, errors::ReleaseCLIError, version};
+use crate::{
+    changelog::{self, Changelog},
+    config,
+    errors::ReleaseCLIError,
+    inputs::get_release_type,
+    release::Release,
+    version
+};
 use chrono::offset::Local;
 
 /// Creates a new release with the given version based on the given version.
-pub fn run(version: String) -> Result<(), ReleaseCLIError> {
+pub fn run(version_option: Option<String>) -> Result<(), ReleaseCLIError> {
     let config = config::load()?;
     let mut changelog = changelog::load(config.clone())?;
 
-    version::parse(version.as_str())?;
+    let version = match version_option {
+        Some(v) => version::parse(v.as_str())?,
+        None => get_release_version(&changelog)?,
```

Comment:
rename to be more verbose
```suggestion
        None => get_next_release_version(&changelog)?,
```